manifests/config.pp

```diff
@@ -1,10 +1,5 @@
 # Configuration for Katello development
 class katello_devel::config {
-
-  # Required until the katello.yml.erb changes to '@variable' syntax
-  include ::katello::params
```

Comment:
I take that back, @ekohl is right we updated katello.yaml properly so this is good to go